docs/api-docs/payments/payments-api-overview.md

```diff
@@ -537,11 +537,11 @@ Yes, checkouts and orders with more than one consignment can use the Payments AP
 
 **Is store credit supported?**
 
-Store credit is not a supported payment method with the Payments API. Store credit can still be used by the shopper on the storefront, part of the control panel, or with the Checkout API.
+Yes, the Payments API supports store credit usage under a few conditions. The store credit payment method is only available for an order when transacting in the store's default currency and when the shopper has an available store credit balance. Store credit is **not** available for orders created by guest shoppers.
```

Comment:
```suggestion
Yes, the Payments API supports the store credit payment method under the following conditions: 
- The shopper is transacting in the store's default currency. 
- The shopper has a positive store credit balance. 

Store credit is _not_ available for orders created by guest shoppers.
```
This is just a suggestion. A list might be more concise, but the way you have it also works. Your call! 

Changed not from bold to italics. Google style guide recommends italics to draw attention to specific words.